src/FLOWVPM_viscous.jl

```diff
@@ -460,16 +460,16 @@ function zeta_direct(sources, targets, zeta::Function)
     for Pi in targets
         for Pj in sources
 
-            dX1 = Pi.X[1] - Pj.X[1]
-            dX2 = Pi.X[2] - Pj.X[2]
-            dX3 = Pi.X[3] - Pj.X[3]
+            dX1 = Pi[1] - Pj[1]
```

Comment:
Use getter and setter functions here instead of indexing

Reply:
Resolved in [f1993eb](https://github.com/byuflowlab/FLOWVPM.jl/pull/13/commits/f1993ebff28329e265be6bd8c09b7cbfeab65610)